### DISCUSSION TITLE
How can I add Chinese translation for this addon?

### DISCUSSION THREAD

**Discussion by 52fisher** [mbnuqw/sidebery#601] [General]
## Description
A good addon should support more languages

**Comment by mbnuqw** [comment]
I've added basic [instruction](https://github.com/mbnuqw/sidebery/blob/v5/CONTRIBUTING.md#translations) for translation contributions and if there are questions I will try to answer them.

**Reply by 52fisher** [reply]
> I've added basic [instruction](https://github.com/mbnuqw/sidebery/blob/v5/CONTRIBUTING.md#translations) for translation contributions and if there are questions I will try to answer them.

Thank you for all that you do!

I‘ve finished the translation of Simplified Chinese (zh-CN). But I found that these dictionary files have no way to distinguish between Simplified Chinese and Traditional Chinese

For example:
This does not conform to the node syntax rules.
```
'btn.create': {
   en: 'Create',
   ru: 'Создать',
   zh-CN: '创建',  //I can only write zh here, but the correct way to write it is zh-CN
  },
```

```
'btn.create': {
   en: 'Create',
   ru: 'Создать',
   zh-TW: '創建 ',  // it should be 創建 in Traditional Chinese
  },
```

```
'btn.create': {
   en: 'Create',
   ru: 'Создать',
   zh: '创建 ',  // This does not distinguish between Simplified Chinese and Traditional Chinese
  },
```

**Reply by mbnuqw** [reply]
Thank you for your work! I already merged it (I should have asked first).
It's actually possible to change the "zh" id to "zh_CN" so I'll update this soon.

**Reply by mbnuqw** [reply]
All changes are commited now and available in https://github.com/mbnuqw/sidebery/releases/tag/v5.0.0b15
